recipe/conda_build_config.yaml

```diff
@@ -40,24 +40,44 @@ rust_default_cc:
   - x86_64-apple-darwin13.4.0-clang     # [linux64 or (osx and x86_64)]
   - arm64-apple-darwin20.0.0-clang      # [linux64 or (osx and x86_64) or (osx and arm64)]
   - clang-cl                            # [linux64 or (osx and x86_64) or win64]
+
+# This should be max(build_platform_ver, cross_target_platform_ver)
 MACOSX_SDK_VERSION:
-  - 11.0
+  - 0                                   # [linux64]
+  - 0                                   # [linux64 or aarch64]
+  - 0                                   # [linux64 or ppc64le]
+  - 10.9                                # [linux64 or (osx and x86_64)]
+  - 11.0                                # [linux64 or (osx and x86_64) or (osx and arm64)]
+  - 0                                   # [linux64 or (osx and x86_64) or win64]
```

Comment:
@isuruf, I added these back. One question: Would you rather we set the SDK version to `10.9` explicitly for osx-64 -> win-64 or implicitly (via `0`)?